components/centraldashboard/webpack.config.js

```diff
@@ -109,6 +123,12 @@ module.exports = {
         ])
     },
     optimization: {
+        minimizer: [new TerserPlugin({
```

Comment:
LMGT's but what does this do?

Reply:
It handles minification. When used for a production build, it also strips out a `dist/public/vendor.bundle.js.LICENSE` file containing all of the license comments from the vendored dependencies.